.github/workflows/integration-test.yml

```diff
@@ -198,16 +198,22 @@ jobs:
       - name: Run tests through docker image
         run: |
           docker run pycytominer:testing pytest
+      # remove tests for production builds below
+      - name: Remove tests from build source
+        if: (github.event_name == 'push'||  github.event_name == 'schedule') && github.ref == 'refs/heads/main'
+        run: |
+          rm -r ./"${{ env.sdist_extracted_name }}/tests"
```

Comment:
This is something that should be done in the Dockerfile rather than in Github Actions. 

Replace [this copy line](https://github.com/cytomining/pycytominer/blob/3286270321f0baac232556f6f16f61870596cb88/build/docker/Dockerfile#L28) with something like this:

```Dockerfile
COPY README.md pyproject.toml poetry.lock ./
COPY pycytominer ./pycytominer
```
The README.md is required because it is referenced in pyproject.toml and otherwise poetry won't install.

And then [here](https://github.com/cytomining/pycytominer/blob/3286270321f0baac232556f6f16f61870596cb88/build/docker/Dockerfile#L35) add:

```Dockerfile
COPY tests ./tests
```